pallets/regions/src/tests.rs

```diff
@@ -287,7 +287,7 @@ fn on_timeout_works() {
 		// failed to decode region_id
 		let mut invalid_get_req = get.clone();
 		invalid_get_req.keys.push(vec![0u8; 15]);
-		assert_noop!(
+		assert_err!(
```

Comment:
I have no idea why the state is modified even if it errors.

Reply:
The reason the state was modified is because there were multiple keys in the request, one of which was correct.